OneSignalSDK/unittest/src/test/java/com/onesignal/MockUserState.java

```diff
@@ -0,0 +1,170 @@
+package com.onesignal;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class MockUserState extends UserState {
+
+    protected MockUserState currentUserState;
+
+    protected final Object syncLock = new Object() {};
+    private String persistKey;
+
+    public JSONObject dependValues, syncValues;
+
+    private static final String[] LOCATION_FIELDS = new String[] { "lat", "long", "loc_acc", "loc_type", "loc_bg", "loc_time_stamp", "ad_id"};
+    private static final Set<String> LOCATION_FIELDS_SET = new HashSet<>(Arrays.asList(LOCATION_FIELDS));
+
+    public MockUserState(String inPersistKey, boolean load) {
+        super(inPersistKey, load);
+        persistKey = inPersistKey;
+        if (load)
```

Comment:
as code style, let keep consistent, if the else block has {} lets put to the if block the {} even if it is a single line